### PR TITLE
[Play] - Final Results - Congratulations Screen 

### PR DESCRIPTION
**Pull request by drewjhart**
This is the base PR of a two part PR for the Final Results pages. This PR layouts out the Congratulations screen that appears prior to the Leaderboard screen.

**PR Structure**
[[Play] - Final Results - Congratulations Screen #595](https://github.com/rightoneducation/righton-app/pull/595)
--> [[Play] - Final Results - Leaderboard Screen #596](https://github.com/rightoneducation/righton-app/pull/596)

**Build**
For testing, I have built and uploaded the two PRs and added some debug buttons so you can still review  the Phase Results pages from [PR593](https://github.com/rightoneducation/righton-app/pull/593). 

You can access this at: [play.rightoneducation.com](play.rightoneducation.com)

**Storybook**
You can access the updated storybook here: 
https://64220218d457f84ba4546eb1-tzmumcnlww.chromatic.com/

**Screenshots**
Screenshot 2023-05-03 at 4 13 48 PM

**Figma Reference**
image
https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?node-id=13554-169669&t=Yc5EYaJ3CJRuiTB7-4

@amarax,
I've pulled the images of the monsters holding the signage from: 
https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?node-id=13554-169669&t=Yc5EYaJ3CJRuiTB7-4

I think we're getting the same issue with the image size normalization as we had previously. I've added a new screenshot to the existing github issue, if that works for you. 
https://github.com/rightoneducation/righton-app/issues/587#issuecomment-1533885790

### CODE REVIEW

**Comment by amarax** [comment]
image
The monsters are not synchronised with the background colour, need to check if I mixed up the monster index.

**Comment by amarax** [comment]
@drewjhart for asset exports in future, please do check whether the assets have been prepared on the [ASSET EXPORT](https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?node-id=11326-334676) page on Figma. These assets have been prepared specially to have the correct export settings, everything else is not guaranteed to have that same scrutiny. If it's missing from the page, you may first use the one directly from the Figma file as a placeholder but it may not be set up with the correct settings yet.

**Comment by drewjhart** [comment]
> image
> 
> The monsters are not synchronised with the background colour, need to check if I mixed up the monster index.

Niec. I've matched the gradients per ASSET EXPORT.

File: play/src/components/finalresults/Congrats.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          <MonsterContainer style={{zIndex:0}}>
+            <Monster
+              src={monsterMap[selectedAvatar].handsup}
+              alt="monster"
+            />
+          </MonsterContainer>
+          <img src={Podium} alt="podium" style={{position: 'absolute', width: '150px', zIndex: -1, top: '100%', marginTop: '-20px'}}/>
+        </Stack>
+        <BottomBox style={{zIndex:1}}> 
+          <GamePlayButtonStyled > View Leaderboard </GamePlayButtonStyled>
```
Comment (maniramezan) [inline play/src/components/finalresults/Congrats.tsx:99]: nit: extra spaces

File: play/src/components/finalresults/Congrats.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+          </Box>
+        <Stack style={{zIndex:0 , display: 'flex', flexDirection: 'column', alignItems: 'center', position: 'relative'}}>
+          <SignCard>
+            { leader ? 
+              <Typography variant="h2" sx={{ textAlign: 'center', whiteSpace: 'pre-wrap'}}>
+                {congratsString}
+              </Typography>
+            :
+              <Typography variant="h6" sx={{ textAlign: 'center'}}>
+               {greatJobString}
```
Comment (maniramezan) [inline play/src/components/finalresults/Congrats.tsx:86]: nit: indent

**Review by maniramezan — APPROVED** [review]

File: play/src/components/finalresults/Congrats.tsx (hunk 1 above)
Comment (drewjhart) [reply play/src/components/finalresults/Congrats.tsx:99]: nice, fixed

**Pull request merged**